test/image/mocks/gl3d_ibm-plot.json

```diff
@@ -196,21 +196,6 @@
                     20.0538,
                     17.7003,
                     15.4519
-                ],
```

Comment:
Note: this extra unused data impacted the `gl3d_ibm-plot.json` (i.e. the undesirable highlighted edge on the right).

Reply:
Interesting... Here's what `contour` does in a similar situation:
![screen shot 2018-11-26 at 7 33 12 pm](https://user-images.githubusercontent.com/2678795/49050670-6bcefc80-f1b2-11e8-8ef6-a5d13900654c.png)
It seems to extrapolate the last differences in x & y. That's a better analog for `surface` than `heatmap`, where both n and n+1 points in x or y compared to the same dimension of z have a clear interpretation.
Doesn't need to be addressed now, but at some point we should probably ensure all our 2D mappings handle this case identically.

Reply:
There is also this related to interpolation. https://github.com/plotly/plotly.js/issues/2812